### PR TITLE
Improvement to bokeh callback mechanism

### PR DESCRIPTION
**Pull request by philippjfr**
Minor improvements to the callback mechanism including an initial delay before executing an event. Recently introduced the same in datashader and seems to work very well. Just tested it on the NYC taxi dataset with 11 million downsampled to 20k points and it's amazingly responsive and works well. Best I think we'll be able to do in terms of user experience.

### CODE REVIEW

**Comment by philippjfr** [comment]
Ready to review/merge.

File: holoviews/plotting/bokeh/callbacks.py [hunk 1]
```diff
@@ -65,6 +65,9 @@ class Callback(param.ParameterizedFunction):
         Avoid running the callback if the callback data is unchanged.
         Useful for avoiding infinite loops.""")
 
+    delay = param.Number(default=200, doc="""
+        Delay when initiating callback events.""")
```
Comment (jlstevens) [inline holoviews/plotting/bokeh/callbacks.py]: Milliseconds I assume?

Comment (philippjfr) [inline holoviews/plotting/bokeh/callbacks.py]: Good point, I'll add it.

**Comment by philippjfr** [comment]
Improved the docstrings now. Ready to merge.

**Comment by jlstevens** [comment]
Merging.

**Pull request merged**